projects/step-frontend/src/lib/modules/execution/services/execution-tree-paging.ts

```diff
@@ -1,7 +1,43 @@
-import { InjectionToken } from '@angular/core';
+import { inject, Injectable, InjectionToken, OnInit } from '@angular/core';
+import { UserService } from '@exense/step-core';
+import { BehaviorSubject } from 'rxjs';
 
-export type ExecutionTreePaging = Record<string, { skip: number }>;
+export type ExecutionTreePagingSetting = Record<string, { skip: number }>;
 
-export const EXECUTION_TREE_PAGING = new InjectionToken<ExecutionTreePaging>('Execution tree paging');
+export const EXECUTION_TREE_PAGING = new InjectionToken<ExecutionTreePagingSetting>('Execution tree paging');
 
-export const EXECUTION_TREE_PAGE_LIMIT = 1000;
+@Injectable()
+export class ExecutionTreePaging {
```

Comment:
```suggestion
export class ExecutionTreePagingService {
```
The file itself should be renamed to `execution-tree-paging.service`